script/release-vm-images

```diff
@@ -34,9 +34,9 @@ source "${ROOT}/script/lib/aws.sh"
 source "${ROOT}/script/lib/util.sh"
 
 UBUNTU_BOX_NAME="ubuntu-trusty-20150204.1" # used in filenames for caching, bump at the same time as UBUNTU_BOX_URL
```

Comment:
This variable needs bumping to invalidate the cache (we could just use `UBUNTU_BOX_SHA` for this).